android/guava/pom.xml

```diff
@@ -106,23 +106,8 @@
       <plugin>
         <artifactId>maven-source-plugin</artifactId>
       </plugin>
-      <!-- TODO(cpovirk): include JDK sources when building testlib doc, too -->
       <plugin>
         <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
Do you happen to have thoughts on whether it might make sense to remove this entirely? I'm guessing so, but I'm not sure.